lib/base-controller.js

```diff
@@ -23,27 +23,42 @@ function getErrorLength() {
   }
 }
 
-BaseController.prototype.getNextStep = function getNextStep(req, res) {
-  var next = Controller.prototype.getNextStep.apply(this, arguments);
+BaseController.prototype.saveValues = function saveValues(req) {
   var forks = (this.options || {}).forks || [];
 
   _.each(forks, function eachFork(fork) {
```

Comment:
I had the entire class es6'd but then linting failed and i wanted to push the concept up. I think es6 should perhaps be another pr for this repo as it diverges from what i'm trying to do here.